.github/workflows/build.yml

```diff
@@ -21,28 +21,22 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3, 8.0]
+                php: [8.0, 7.4]
                 symfony: [^4.4, ^5.2]
-                sylius: [~1.8.0, ~1.9.0, ~1.10.0]
-                node: [10.x]
-                mysql: [5.7]
+                sylius: [~1.9.0, ~1.10.0, ~1.11.0]
+                node: [14.19]
+                mysql: [8.0]
```

Comment:
Why? Afaik Sylius supports also MySQL 5.7 so it's safer to use the lower version.

Reply:
Reverted to 5.7